Bottom: 8ca28906d37b1f27815240b897a57f3d1772dede
Top:    ba6ced8fbb2dc03f7a0c738b53fb5fb5b9aec815
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 16:11:31 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
